content/applications/inventory_and_mrp/plm/manage_changes.rst

```diff
@@ -0,0 +1,9 @@
+=================
+Change management
+=================
+
+.. toctree::
+   :titlesonly:
+
+   manage_changes/eco_type
+   manage_changes/engineering_change_orders
```

Comment:
would switch the order of these since ECO doc introducing the concept would ideally come first.
fwiw doc content is **not** organized alphabetically in English.